tests/test_benchmark/test_benchmark.py

```diff
@@ -10,7 +10,7 @@
 from sentence_transformers import SentenceTransformer
 
 import mteb
-from mteb.benchmarks import Benchmark
+from mteb.benchmarks.benchmarks import Benchmark
```

Comment:
(Missing this in the 1st round, apologies) How come this is needed when `mteb/benchmarks.__init__.py` has the following?

```
from mteb.benchmarks.benchmarks import *
```

Reply:
it is not needed. I have reverted it :)